code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -33,19 +33,19 @@
 	var/need_mob_update = FALSE
 	switch(affected_mob.stat)
 		if(CONSCIOUS) //bad
-			thou_shall_heal = death_is_coming/50
-			need_mob_update += affected_mob.adjustOxyLoss(2 * REM * seconds_per_tick, TRUE, required_biotype = affected_biotype, required_respiration_type = affected_respiration_type)
+			thou_shall_heal = min(death_is_coming/20, 3)
```

Comment:
this is 3 damage max, not min
should be
`thou_shall_heal = max(death_is_coming/20, 3)`

Reply:
Duh, that's a good catch, I put that on after testing the other vals and realizing I didn't want it scaling off to 1/20 at 99 health or something - thanks

Reply:
still here

Reply:
fixed somehow i missed this